cmd/kubeadm/app/apis/kubeadm/v1beta4/doc.go

```diff
@@ -24,7 +24,7 @@ limitations under the License.
 //
 // A list of changes since v1beta3:
 //
-//   - Support custom environment variables in control plane components under `ClusterConfiguration`.
+//   - Support custom environment variables in control plane components under ClusterConfiguration.
```

Comment:
I'm just asking, should we remove the quotes here? It looks different where other place, such as [L33](https://github.com/kubernetes/kubernetes/pull/126628/files#diff-5c925db27fad81e7d11666a4fe0012055a00ebf625f2e7327b20f846ef54d586R33)

Reply:
Updated. 

Only L39 fixed a doc bug, and others are adding '`' for types like `InitConfiguration`, `JoinConfiguration`, `ResetConfiguration` and `UpgradeConfiguration`.